proto/regen/group/v1alpha1/types.proto

```diff
@@ -0,0 +1,266 @@
+syntax = "proto3";
+
+package regen.group.v1alpha1;
+
+option go_package = "github.com/regen-network/regen-ledger/x/group";
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+import "cosmos_proto/cosmos.proto";
+import "google/protobuf/any.proto";
+
+// Member represents a group member with an account address,
+// non-zero power and a comment.
+message Member {
+
+    // address is the member's account address.
+    string address = 1;
+    
+    // power is the member's power that should be greater than 0.
+    string power = 2;
+    
+    // comment is the member's comment.
+    string comment = 3;
```

Comment:
Maybe we should rename all these comments to `metadata`? And have them be `bytes` link in `x/ecocredit`? Then they could maybe be linked to ID's in `x/data`? Wdyt?

Reply:
sounds good!

Reply:
The `Params` `MaxCommentLength` doesn't seem to be relevant anymore in this context, should we just get rid of it?

Reply:
How about just `MaxMetadatLength`?